src/Core/PersistedQueries.Abstractions/IWriteStoredQueries.cs

```diff
@@ -0,0 +1,19 @@
+using HotChocolate.Execution;
+using System.Threading.Tasks;
+
+namespace HotChocolate.PersistedQueries
+{
+    /// <summary>
+    /// A tool for storing queries to some persistence medium.
+    /// </summary>
+    public interface IStoreQueries
```

Comment:
Should we rename than to IWriteStoredQueries .... just to have some symmetry between the names?